test/Generator/TestAsset/ParameterClass.php

```diff
@@ -75,7 +75,7 @@ public function defaultConstant($con = self::FOO)
 
     }
 
-    public function defaultObjectEqualsNullAndNotOptional(\stdClass $a = null, $b)
+    public function defaultObjectEqualsNullAndNotOptional(?\stdClass $a, $b)
```

Comment:
hm..., this method name seems no longer represent to display, probably just rename the method

Reply:
Okay. How does name below name looks like ?

`public function handleNullableObject(?\stdClass $a, $b)`

Reply:
`nullableAndRequired` is fine for me